kratos/utilities/math_utils.h

```diff
@@ -1,50 +1,15 @@
-/*
```

Comment:
My first and most important comment is that this function is used in a lot of places, most of which have no tests.
for this reason if you like to change (and i am positive you do)
you should

STEP1 - create a test written in c++ that tests all of the methods
STEP2 - change it
STEP3 - verify that the tests still run
STEP4 - push the change.

I can not accept a change to this without u going through this procedure... but i hope it is not TOO time consuming to do so.

@roigcarlo could u explain to @loumalouomega how to create and run a test in c++?